### DISCUSSION TITLE
What would cause Apex Rollup to stop working?

### DISCUSSION THREAD

**Discussion by jdids247** [jamessimone/apex-rollup#553] [Q&A]
I know this is really open ended, but we have a situation where Apex Rollup has just stopped working altogether. 

We use CMDT and a trigger to run our rollups. 

This is specifically in our Sandboxes. Nothing has changed on any configuration. I've gone as far as completely uninstalling and reinstalling the application, enabling custom settings, setting the Org defaults, etc....not even a CMDT rollup from the GUI works. 

I've tried installing Nebula to see what could be happening, but even that isn't pushing data. 

Unsure where to go from here or how to generate logs

EDIT: Using 1.6.9 in Sandboxes and using 1.6.2 in Production

**Comment by jamessimone** [comment]
That's a great question, and unfortunately I'm not totally sure. What I would recommend is setting `Is Rollup Logging Enabled` on the Rollup Control record to true, performing an action that _should_ be making a rollup fire with your Salesforce Developer Console open, and investigating some of the debug log files that get generated in the Log pane of the developer console as a result. If you toggle the `Debug Only` checkbox there, you _should_ see messages from Apex Rollup being generated - and hopefully those messages will help us to diagnose what's happened!

**Reply by jdids247** [reply]
Wonderful information on debugging. Newish to Apex and development so every little nugget helps.

The debug logs are showing that the rollup was not processed and I'm seeing some other logs that aren't making much sense
![image](https://github.com/jamessimone/apex-rollup/assets/110419431/1b209ee6-3025-48e7-b472-6515b0e97e94)

Not sure where to go from here.
